demos/crossroad_camera_demo/cpp/main.cpp

```diff
@@ -311,9 +307,7 @@ int main(int argc, char* argv[]) {
                 }
             }
             framesProcessed++;
```

Comment:
Now the number of written frames is counted in `LazyVideoWriter`, so the `framesProcessed` variable should be removed from demos

Reply:
Removed

Reply:
`crossroad_camera_demo` is left

Reply:
Removed